action.yml

```diff
@@ -24,23 +24,20 @@ inputs:
     description: "Enabled Nuget Push to Package Registry."
     required: false
     default: "false"
-  nuget_package_name:
-    description: "Creates the nuget package with this name.Used only when nuget_push_enabled is true."
-    required: false
-  nuget_project_path:
-    description: "Path to the Nuget Project File (.csproj).Used only when nuget_push_enabled is true.Required if nuget_push_enabled is true."
-    required: false
   container_push_enabled:
     description: "Enable Build and Push Container Image"
     required: false
     default: "true"
-  github_token:
-    description: "Github token"
+  nuget_pull_token:
+    description: "GitHub token with repo read permissions for pulling NuGet packages"
     required: true
+  nuget_push_token:
+    description: "GitHub token with package write permissions for pushing NuGet packages"
+    required: false
 
 
 runs:
   using: "docker"
-  image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:v0.1.2
+  image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:sha-f89e2903df9f056356c9018c99af7774025624da
```

Comment:
@vibindaniel should I set this as v0.1.3 and then tag upon approval/merge?

Reply:
Yes, once you do that you can merge it.